### PR TITLE
more swe examples

### PR DESCRIPTION
**Pull request by nicksnyder**

### CODE REVIEW

File: job-descriptions/software-engineer.md [hunk 1]
```diff
@@ -1,46 +1,75 @@
...
+- A track record of delivering high-quality products with attention to scalability and UX.
+- Strong web/JavaScript/TypeScript/Go fundamentals.
+- Experience working with APIs and distributed systems.
+- Passion for the craft of software development and good engineering practices.
 
-- Design and build multi-tier web & desktop applications using Go & TypeScript
-- Combine system metrics, user feedback, and intuition to plan improvements
-- Create simple and performant APIs and services based on multiple underlying data sources
-- Lead and mentor other team members while driving technical roadmaps and decisions for various projects
+### Example projects
```
Comment (nicksnyder) [inline job-descriptions/software-engineer.md]: @felixfbecker @KattMingMing @ijsnow @keegancsmith @slimsag @ggilmore 

Let me know if you can think of examples that are better than the ones I already have

Comment (KattMingMing) [reply job-descriptions/software-engineer.md]: These look great - There could also be open tickets on the browser extension repository that would be good for somebody interviewing to tackle as well. Once CXP is in production we should update the example to include creating an extension.

Comment (dadlerj) [reply job-descriptions/software-engineer.md]: Actually Nick just added those to the projects list yesterday: https://docs.google.com/document/d/1kIxMMFTNlCQ2ofl2aq9K3GFFeGkl5JgSdpR-6mor_Ig/edit#! This is just the job description to give examples to people

**Comment by nicksnyder** [comment]
Going to merge this now, but feel free to continue to make suggestions.

**Pull request merged**